content/spin/managing-templates.md

```diff
@@ -83,16 +83,19 @@ $ spin templates uninstall redis-befunge
 
 ## Upgrading Templates
 
-When you upgrade Spin, you will typically want to upgrade your templates to match.  This means new applications and components will get dependencies that match the Spin version you are using.  To do this, run `spin templates upgrade`:
+<!-- @searchTerm "update templates to point to canary SDKs" -->
```

Comment:
This is general upgrading and would not typically pick up canary SDKs. For canary you’d usually update the SDK reference in the generated app rather than upgrading the templates.